jubatus/server/framework/server_base.hpp

```diff
@@ -76,18 +76,22 @@ class server_base {
   }
 
   uint64_t last_saved_sec() const {
+    jubatus::util::concurrent::scoped_rlock lock(status_mutex_);
     return last_saved_.sec;
   }
 
   const std::string& last_saved_path() const {
```

Comment:
Is it safe to return pointer of the protected resource? (same for `last_loaded_path`)